docs/en/ingest-management/fleet-limitations.asciidoc

```diff
@@ -20,16 +20,15 @@ This feature has additional limitations at the current time:
 *   Requires internet access for {kib} to download integration packages
 *   No support for advanced {beats} settings like multiline, processors, and so
```

Comment:
Should you remove processors as an example and say something else here? I'm assuming processors are supported since they are shown in all the variable/condition examples. :-)

Reply:
is still not expose in fleet so I'll keep it.